submissions/damisiMahmood/html-forms/contactPage.html

```diff
@@ -0,0 +1,61 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <header><img src="C:\Users\peter\Desktop\Dufuna-CodeCamp20\submissions\damisiMahmood\html\images/fashionlogo.png" alt="All Things Fashion">
+            <h1> I-DAMSELS ELITE SHOPPING </h1> 
+        </header>
+        <nav>
+            <ul>
+                <li><a href="#"> HOME</a></li> 
+                <li><a href="#">ABOUT</a></li> 
+                <li><a href="#">CATGORIES</a></li> 
+                <li><a href="#">BRANDS</a></li> 
+                <li><a href="#">SIGN OUT</a></li>
+            </ul>
+        </nav>
+        <fieldset>
+            <legend><h1>Contact Us</h1></legend>
+        <form action="#" method="post" enctype="multipart/form-data">
+            <label for="fullname">Full Name</label> <br>
+            <input type="text" name="fullname" id="fullname" placeholder="First name  Last name" size="40" maxlength="30"/><br>
+            <br>
+            <label for="email">Email</label> <br>
+            <input type="email" name="email" id="email" placeholder="Enter your email address"size="40" maxlength="30"/> <br>
+            example@example.com <br>
+            <br>
+            <label for="subject">Subject</label> <br>
+            <input type="text" name="subject" id="Subject" size="40" maxlength="30"/><br> <br>
+            <label for="message">Message</label> <br>
+            <textarea rows="7" cols="50" name="Message" maxlength="300"></textarea>
+            <br>
+            <br>
+
+            <button type="submit">Submit</button>
+        </form>
+        </fieldset>
+        <br>
+    <footer> 
+        <section>
+            <img src="C:\Users\peter\Desktop\Dufuna-CodeCamp20\submissions\damisiMahmood\html\images/brandslogo.png" width="1200" height="400" alt="OUR BRANDS">
```

Comment:
This is not this best way to target a path. Just imagine I clone your work. Of course, my system won't have this path `C:\Users\peter\`. Kindly, used `<img src="../html/images/brandslogo.png" width="1200" height="400" alt="OUR BRANDS">`

Reply:
This has been reviewed. Kindly review. Thank you